projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/KubernetesResources.java

```diff
@@ -8,30 +8,34 @@
 import com.vmware.taurus.service.KubernetesService;
 import lombok.AllArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 @Component
 @AllArgsConstructor
 public class KubernetesResources {
-  // TODO: clients may need to configure those - consider making them configurable.
+  @Value("${datajobs.deployment.initContainer.resources.requests.cpu}")
+  private String dataJobInitContainerCpuRequests;
 
-  // Data jobs Default resource configuration for the init container only.
-  public static final String DATA_JOB_INIT_CONTAINER_CPU_REQUESTS = "100m";
-  public static final String DATA_JOB_INIT_CONTAINER_MEMORY_REQUESTS = "100M";
+  @Value("${datajobs.deployment.initContainer.resources.requests.memory}")
+  private String dataJobInitContainerMemoryRequests;
 
-  public static final String DATA_JOB_INIT_CONTAINER_CPU_LIMITS = "100m";
```

Comment:
the limits were priorly 100, and now changed by default to 500?
why not keeping the default values, yet have the change so they are configurable?

Reply:
I think it will be a problem for more and more customers in the future. So it is a good change especially since it is only for `limits` not for `requests`.